library/datetime/to_time_spec.rb

```diff
@@ -1,6 +1,25 @@
 require File.expand_path('../../../spec_helper', __FILE__)
 require 'date'
 
-describe "DateTime#to_time" do
-  it "needs to be reviewed for spec completeness"
+ruby_version_is "2.4" do
+  describe "DateTime#to_time" do
+    it "yields a new Time object" do
+      DateTime.now.to_time.should be_kind_of(Time)
+    end
```

Comment:
This spec can be outside the `2.4` block, DateTime#to_time already existed before but it did not preserve the timezone.

Reply:
If I remove it from the 2.4 block, will it not fail on prior versions because of the lack of timezone preservation?

Reply:
Yes, that's why I would only move this example out of the 2.4 block.